Description: Fix desktop file as Freedesktop.org's per spec.
Author: Chow Loong Jin <hyperair@gmail.com>
Author: James Cowgill <james410@cowgill.org.uk>
Last-Update: 2014-04-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Runtime/codelite.desktop.template
+++ b/Runtime/codelite.desktop.template
@@ -1,9 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=codelite
+Name=CodeLite
 Exec=codelite %f
 Icon=%%PREFIX%%/share/codelite/images/cubes.png
 Terminal=false
 Type=Application
 Categories=Development;
 StartupNotify=true
+Keywords=IDE;development;programming;
